### DISCUSSION TITLE
Getting Started in the Community

### DISCUSSION THREAD

**Discussion by mattchristenson** [oqtane/oqtane.framework#4522] [General]
Hi everyone!

I'm about to embark on a deep dive into Oqtane. What are the MUST HAVE resources that you recommend looking at?

For example - I've found this series on YouTube for Oqtane module development - although it was 4 years ago - is this still mostly up-to-date and a good place for a technical overview of the framework and platform?  https://www.youtube.com/watch?v=JPfUZPlRRCE&list=PLYhXmd7yV0elLNLfQwZBUlM7ZSMYPTZ_f

That didn't seem to be referenced in the docs - at least it wasn't immediately obvious to me, I found it by searching YouTube., I wonder if there are any other resources that are worth checking out?

I see that we're using this space for general community discussion - there's one place where I heard mention of a weekly developers meeting - where can I find information about that?  Is there also a chat on the topic, or is this the place to be?

Looking forward to getting to know community members!

Matt Christenson

**Comment by leigh-pointer** [comment]
Hi @mattchristenson, welcome!

Although the series is somewhat outdated, it still offers valuable insights. Oqtane has the ability to scaffold a basic module, which is a great place to start. You can do this directly from the source code version of the framework. After logging in, navigate to the Admin Dashboard, click on Module Management, and select the option to create a new module. Oqtane will then scaffold your module in the appropriate location.

Feel free to join the Discord community for further support and discussion: [Discord Server](https://discord.gg/ht65VqdG).

**Comment by leigh-pointer** [comment]
https://discord.gg/BnPny88avK
________________________________
